src/Sulu/Bundle/MediaBundle/Tests/Functional/Controller/MediaPreviewControllerTest.php

```diff
@@ -0,0 +1,222 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\MediaBundle\Tests\Functional\Controller;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Sulu\Bundle\MediaBundle\Entity\Collection;
+use Sulu\Bundle\MediaBundle\Entity\CollectionMeta;
+use Sulu\Bundle\MediaBundle\Entity\CollectionType;
+use Sulu\Bundle\MediaBundle\Entity\File;
+use Sulu\Bundle\MediaBundle\Entity\FileVersion;
+use Sulu\Bundle\MediaBundle\Entity\FileVersionMeta;
+use Sulu\Bundle\MediaBundle\Entity\Media;
+use Sulu\Bundle\MediaBundle\Entity\MediaType;
+use Sulu\Bundle\TestBundle\Testing\SuluTestCase;
+use Symfony\Component\HttpFoundation\File\UploadedFile;
+
+class MediaPreviewControllerTest extends SuluTestCase
+{
+    /**
+     * @var EntityManagerInterface
+     */
+    private $em;
+
+    /**
+     * @var MediaType
+     */
+    private $imageType;
+
+    /**
+     * @var MediaType
+     */
+    private $videoType;
+
+    /**
+     * @var Collection
+     */
+    private $collection;
+
+    public function setUp(): void
+    {
+        $this->em = $this->getEntityManager();
+
+        $this->collection = new Collection();
+
+        $collectionType1 = new CollectionType();
+        $collectionType1->setId(1);
+        $collectionType1->setName('Default Collection Type');
+        $collectionType1->setDescription('Default Collection Type');
+
+        $collectionType2 = new CollectionType();
```

Comment:
is this used anywhere? 🙈

Reply:
Yes, although it is quite hidden... [The `SystemCollectionManager` creates a `Collection` with the `CollectionType` with the hardcoded ID 2](https://github.com/sulu/sulu/blob/e7ce3d7/src/Sulu/Component/Media/SystemCollections/SystemCollectionManager.php#L273). :see_no_evil: